### DISCUSSION TITLE
Create separate lists when the list contains both legal and illegal domains

### DISCUSSION THREAD

**Discussion by ghost** [blocklistproject/Lists#263] [Ideas]
I was thinking it might be a good thing to split some of the lists into legal and illegal lists, for example

- Crypto
-- Crypto Legal
-- Crypto Illegal
- Drugs
-- Legal Drugs (Rx drugs, for sure)
-- Illegal Drugs (understand this is US-based)
- Gambling (this would be tricky)
-- Legal Gambling
-- Illegal Gambling

Maybe Torrent.  The reason is that there are good Crypto sites for Monero, Bitcoin, etc., as there are good sites for legal Rx drugs, gambling, and on the outside, Torrent sites.  (I'm ambivalent on Torrent.)

I'd be interested in your thoughts.

**Comment by blocklistproject** [comment]
This is a very interesting idea... The issue I see in doing this starts with... Illegal where? Let's take a simple if not controversial topic, marijuana. Where would this fall on the lists because in the states we can't even as a country decide if this is legal or illegal. 

I like the idea and would not mind at all hearing the community's idea on how this might be accomplished.
